0006-cmx2cdp-protocol/README.md

```diff
@@ -1,84 +1,102 @@
 ---
 name: CMX to CDP protocol
 shortname: 6/CMX2CDP
-status: raw
-editor: Andrew Pasika <apasika@cybervisiontech.com>
+status: draft
```

Comment:
The status only denotes how widely the protocol is used (raw=not used, draft=prototypes, stable=relied by 3rd parties, deprecated=only old software should use/should be replaced, retired=not used/replaced).

I wouldn't say that raw protocols shouldn't be reviewed.